docs/pages/enroll-resources/database-access/auto-user-provisioning/postgres.mdx

```diff
@@ -75,10 +87,6 @@ the database, which will be created automatically if it doesn't exist.
 
 (!docs/pages/includes/database-access/auto-user-provisioning/db-definition-default-dbname.mdx protocol="postgres" uri="localhost:5432" default="the same database that the user is accessing" !)
 
-<Admonition type="warning" title="Procedure Privileges in PostgreSQL 15+">
-(!docs/pages/includes/database-access/auto-user-provisioning/postgres15-grant-create.mdx!)
-</Admonition>
```

Comment:
`GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA schema1, schema2, schema3 TO "teleport-admin";`

This shouldn't be necessary any more after @gabrielcorado 's [change](https://github.com/gravitational/teleport/pull/44255) that moved procedures to `pg_temp`